livestyled/models/form.py

```diff
@@ -28,9 +63,9 @@ def __init__(
         self.allow_update = allow_update
         self.refresh_on_success = refresh_on_success
         if fields:
-            self.fields = [FormField(**ff) for ff in fields]
+            self.fields = [FormField(**field) for field in fields]
         if translations:
-            self.fields = [FormTranslation(**ft) for ft in translations]
+            self.fields = [FormTranslation(**translation) for translation in translations]
```

Comment:
So just found out this will raise an exception since translations isn't an Iterable and the for comprehension will raise the exception. 

Something like this will fix it = 
```
-> translations and [FormTranslation(**translation) for translation in translations]
```

also to get rid of the if statement in the constructor.

Reply:
Translations is an array, I'm not sure what you mean? The tests for this pass.